consensus/spos/bls/subroundStartRound.go

```diff
@@ -121,6 +127,12 @@ func (sr *subroundStartRound) initCurrentRound() bool {
 		return false
 	}
 
+	currentRound := sr.RoundHandler().Index()
```

Comment:
There are more 4 more places below in this method where you can replace `sr.RoundHandler().Index()` with `currentRound` variable.

Reply:
right, will be replaced in the next PR